gtda/time_series/tests/test_preprocessing.py

```diff
@@ -44,6 +44,15 @@ def test_resampler_transform(period, expected):
     assert_almost_equal(resampler.fit_transform(signal_array), expected)
 
 
+@pytest.mark.parametrize("period, expected",
+                         [(2, signal_resampled_2),
+                          (4, signal_resampled_4)])
+def test_resampler_resample(period, expected):
+    resampler = Resampler(period=period)
+    _ = resampler.fit_transform(signal_array)
```

Comment:
I think it's enough (and would be clearer) to only use fit here, since resample does not necessitate a run of transform to work.